src/index.js

```diff
@@ -9,3 +9,5 @@ export { default as Search } from './Search/index.tsx'
 export { default as ListCard } from './ListCard/index.tsx'
 export { default as AppBar } from './AppBar/index.tsx'
 export { default as Map } from './Map/index.tsx'
+export * from './Heading/index.tsx'
+export { default as Section } from './Section/index.tsx'
```

Comment:
why do we use this file?

Reply:
It's the default entrypoint for the lib. Otherwise you won't be able to use it in the frontend